internal/components/components.go

```diff
@@ -209,6 +210,7 @@ func Components() (otelcol.Factories, error) {
 		oidcauthextension.NewFactory(),
 		sigv4authextension.NewFactory(),
 		zpagesextension.NewFactory(),
+		ackextension.NewFactory(),
```

Comment:
Nit: The extensions are in alphabetical order so far, this should be moved up to match existing order.